miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/AdBridgeDispatcher.kt

```diff
@@ -0,0 +1,105 @@
+package com.rakuten.tech.mobile.miniapp.js
+
+import com.google.gson.Gson
+import com.rakuten.tech.mobile.miniapp.ads.MiniAppAdDisplayer
+import com.rakuten.tech.mobile.miniapp.ads.isAdMobProvided
+
+@Suppress("TooGenericExceptionCaught", "LongMethod")
+internal class AdBridgeDispatcher {
+    private lateinit var bridgeExecutor: MiniAppBridgeExecutor
+    private lateinit var adDisplayer: MiniAppAdDisplayer
+    private var isAdMobEnabled = false
+
+    fun setAdMobDisplayer(adDisplayer: MiniAppAdDisplayer) {
```

Comment:
It means, we need to update the related change in [USERGUIDE](https://github.com/rakutentech/android-miniapp/blob/master/miniapp/USERGUIDE.md#admob)?

Reply:
No, there is nothing change in user implementation.